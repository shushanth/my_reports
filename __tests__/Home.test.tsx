import "@testing-library/jest-dom";
import { render, screen } from "@testing-library/react";
import Home from "@/app/page";

describe("Home", () => {
  it("renders component", () => {
    render(<Home />);
    const heading = screen.queryByTestId("home_page--component");
    expect(heading).toBeDefined();
  });

  it("renders component unchanged", () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});
