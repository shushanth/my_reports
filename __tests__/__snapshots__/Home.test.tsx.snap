// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders component unchanged 1`] = `
<div>
  <div
    data-testid="home_page--component"
  >
    Home
    <div
      class="button button_primary button_medium button--straight"
    >
      <button
        class=""
      >
        My Reports
      </button>
    </div>
  </div>
</div>
`;
